Description: Upstream changes introduced in version 0.9.6-0oneiric
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ros-fuerte-ompl (0.9.6-0oneiric) oneiric; urgency=high
 .
   * autogenerated
 .
 The person named in the Author field signed this changelog entry.
Author: Ioan Sucan <isucan@willowgarage.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ ros-fuerte-ompl-0.9.6/stack.yaml
@@ -0,0 +1,15 @@
+Catkin-ProjectName: ompl
+Version: 0.9.6
+
+Depends: cmake,
+    libgtest-dev,
+    boost
+
+Description: OMPL is a free sampling-based motion planning library.
+
+Author: Kavraki Lab
+Maintainer: Ioan Sucan <isucan@willowgarage.com>
+Homepage: http://ompl.kavrakilab.org
+
+Catkin-CopyrightType:  willowgarage
+Catkin-DebRulesType: cmake
--- /dev/null
+++ ros-fuerte-ompl-0.9.6/catkin.conf
@@ -0,0 +1,3 @@
+[catkin]
+	upstream = http://ompl.hg.sourceforge.net:8000/hgroot/ompl/ompl
+	upstreamtype = hg
